packaging/windows/variables.wxi

```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0"?>
 <Include xmlns="http://schemas.microsoft.com/wix/2006/wi">
+  <?if $(var.ReleaseSuffix) ~= "beta" OR $(var.ReleaseSuffix) ~= "rc1" OR $(var.ReleaseSuffix) ~= "rtm" OR $(var.ReleaseSuffix) ~= ""?>
+  <?define BuildType="Release"?>
```

Comment:
ok for now, but we'll need to keep updating this. Let's make this a boolean field that denotes we're in the release/prod train. Simple reg key?